src/input.rs

```diff
@@ -556,10 +538,14 @@ impl<'a, A: ActionContext + 'a> Processor<'a, A> {
         let launcher = self.mouse_config.url.launcher.as_ref()?;
         let mut args = launcher.args().to_vec();
         args.push(text);
-
-        match Command::new(launcher.program()).args(&args).spawn() {
-            Ok(_) => debug!("Launched: {} {:?}", launcher.program(), args),
-            Err(_) => warn!("Unable to launch: {} {:?}", launcher.program(), args),
+        let spawned=start_daemon(launcher.program(),&args);
+        match spawned {
```

Comment:
This can also be rewritten to `match start_daemon(launcher.program(), &args) {`.